lib/facter/puppet_environmentpath.rb

```diff
@@ -0,0 +1,21 @@
+# This facter fact returns the value of the Puppet environmentpath. It is adapted
+# from the puppetlabs-stdlib puppet_vardir fact.
```

Comment:
Could you submit that to puppetlabs-stdlib? If they don't take it, it can still get included here.